lib/ret/hub.ex

```diff
@@ -37,6 +37,7 @@ defmodule Ret.Hub do
     field(:entry_mode, Ret.Hub.EntryMode)
     belongs_to(:scene, Ret.Scene, references: :scene_id)
     has_many(:web_push_subscriptions, Ret.WebPushSubscription, foreign_key: :hub_id)
+    belongs_to(:account, Ret.Account, references: :account_id)
```

Comment:
not totally sure, but we might want to consider naming this `created_by_account_id` since hubs are going to have a variety of semantic relationships to users -- eg there's already the concept of "owner" and "creator" implicitly so naming it this way would make it clear what exactly is being recorded